### ISSUE TITLE
[System.Net.Http.Json] System.Net.Http.Json - Undoc 6.0 APIs

### ISSUE THREAD

**Issue by karelz** [dotnet/runtime#58750]
Area owners: @Jozkee cc @eiriktsarpalis @steveharter @layomia

The following APIs were introduced in 6.0 and did not get documented in triple slash. They will need to be manually documented directly in dotnet-api-docs, and must target the net6-rc1 branch. This needs to be done before Tuesday September 14th so we have enough time to generate the IntelliSense package that will be shipped to GA.

| Undoc APIs |
|--|
| [M:System.Net.Http.Json.HttpClientJsonExtensions.GetFromJsonAsync(System.Net.Http.HttpClient,System.Uri,System.Type,System.Text.Json.Serialization.JsonSerializerContext,System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpClientJsonExtensions.xml)                                                |
| [M:System.Net.Http.Json.HttpClientJsonExtensions.GetFromJsonAsync(System.Net.Http.HttpClient,System.String,System.Type,System.Text.Json.Serialization.JsonSerializerContext,System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpClientJsonExtensions.xml)                                             |
| [M:System.Net.Http.Json.HttpClientJsonExtensions.GetFromJsonAsync\`\`1(System.Net.Http.HttpClient,System.String,System.Text.Json.Serialization.Metadata.JsonTypeInfo{\`\`0},System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpClientJsonExtensions.xml)                                                 |
| [M:System.Net.Http.Json.HttpClientJsonExtensions.GetFromJsonAsync\`\`1(System.Net.Http.HttpClient,System.Uri,System.Text.Json.Serialization.Metadata.JsonTypeInfo{\`\`0},System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpClientJsonExtensions.xml)                                                    |
| [M:System.Net.Http.Json.HttpClientJsonExtensions.PostAsJsonAsync\`\`1(System.Net.Http.HttpClient,System.String,\`\`0,System.Text.Json.Serialization.Metadata.JsonTypeInfo{\`\`0},System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpClientJsonExtensions.xml)                                              |
| [M:System.Net.Http.Json.HttpClientJsonExtensions.PostAsJsonAsync\`\`1(System.Net.Http.HttpClient,System.Uri,\`\`0,System.Text.Json.Serialization.Metadata.JsonTypeInfo{\`\`0},System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpClientJsonExtensions.xml)                                                 |
| [M:System.Net.Http.Json.HttpClientJsonExtensions.PutAsJsonAsync\`\`1(System.Net.Http.HttpClient,System.String,\`\`0,System.Text.Json.Serialization.Metadata.JsonTypeInfo{\`\`0},System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpClientJsonExtensions.xml)                                               |
| [M:System.Net.Http.Json.HttpClientJsonExtensions.PutAsJsonAsync\`\`1(System.Net.Http.HttpClient,System.Uri,\`\`0,System.Text.Json.Serialization.Metadata.JsonTypeInfo{\`\`0},System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpClientJsonExtensions.xml)                                                  |
| [M:System.Net.Http.Json.HttpContentJsonExtensions.ReadFromJsonAsync(System.Net.Http.HttpContent,System.Type,System.Text.Json.Serialization.JsonSerializerContext,System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpContentJsonExtensions.xml)                                                       |
| [M:System.Net.Http.Json.HttpContentJsonExtensions.ReadFromJsonAsync\`\`1(System.Net.Http.HttpContent,System.Text.Json.Serialization.Metadata.JsonTypeInfo{\`\`0},System.Threading.CancellationToken)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System.Net.Http.Json/HttpContentJsonExtensions.xml)                                                           |

<details>
<summary>Old instructions</summary>

The APIs in the table below were introduced in .NET 6.0 and are showing up as undocumented in the dotnet-api-docs repo.

Some of them are already documented in triple slash comments in source. Area owners, please: 

1. Verify if there are any APIs from the table have not been documented in source. If you find any, document them in triple slash and submit a PR to runtime to commit the documentation.
2. Clone the dotnet-api-docs repo and port the triple slash comments by running this [DocsPortingTool](https://github.com/carlossanlop/DocsPortingTool) command:

```
DocsPortingTool \
    -Direction ToDocs \
    -Docs %REPOS%\dotnet-api-docs\xml \
    -Intellisense %REPOS%\runtime\artifacts\bin\ \
    -IncludedAssemblies System.Net \
    -IncludedNamespaces System.Net \
    -Save true
```

3. Submit a PR to dotnet-api-docs with the ported documentation.

</details>

**Comment by carlossanlop** [comment]
@steveharter @layomia Friendly reminder that today is the last day to get this documentation merged into the net6-rc1 branch in dotnet-api-docs so it gets included in the intellisense of the .NET 6.0 release.

**Comment by karelz** [comment]
Fixed in PR https://github.com/dotnet/dotnet-api-docs/pull/7179
